<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei UI</family>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startRec_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Rec </string>
    </property>
   </widget>
   <widget class="QPushButton" name="replayRec_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>95</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay Rec</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="instant_run">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Instant Run</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="hotkey_setting">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="rec_events_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>510</width>
      <height>350</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="repeat_spec_times">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="repeat_until_stop">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat Until Stopped</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="repeat_times_spinBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>times</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
